src/parser.rs

```diff
@@ -56,9 +67,10 @@ impl Material {
     pub fn weight(&self) -> Option<f32> {
         let w = self.get_f32("_weight");
 
-        if let Some(w) = w && (w < 0.0 || w > 1.0)
-        {
-            debug!("_weight observed outside of range of [0..1]: {}", w);
+        if let Some(w) = w {
+            if w < 0.0 || w > 1.0 {
```

Comment:
If you're going to change unrelated code, please do it in a separate PR

Reply:
Yep, not sure if this is still a problem in the latest main branch, will check and see. It was preventing me from compiling at all, but yeah I probably shouldn't have included in the PR :)